common.mk

```
@@ -68,6 +68,11 @@ PRODUCT_PACKAGES += \
 # Force building a recovery image: Needed for OTA packaging to work since Q
 PRODUCT_BUILD_RECOVERY_IMAGE := true
 
+ifneq ($(filter 4.14, $(SOMC_KERNEL_VERSION)),)
```

```suggestion
ifeq ($(SOMC_KERNEL_VERSION), 4.14)
```

Work smarter, not harder.